package-lock.json

```diff
@@ -1,8 +1,2310 @@
 {
   "name": "MovieCollections",
   "version": "1.0.0",
```

Comment:
czemu zmieniłaś ten plik?

Reply:
Pociagnalam pull z maina zanim ci przeslalam. Ta zmiana to Filip 5 dni temu przez PR #35